package.json

```diff
@@ -26,7 +26,7 @@
     "bootstrap": "^3.3.7",
     "bootstrap-daterangepicker": "^2.1.24",
     "bootstrap-tour": "^0.12.0",
-    "enketo-core": "^4.41.0",
+    "enketo-core": "https://github.com/enketo/enketo-core.git#master",
```

Comment:
Obviously we can't commit it like this - and commit to enketo-core could break our install. I assume this was left in by accident. If we have to wait for them to do the release that's fine by me - they're usually pretty quick!

Reply:
Yeah, waiting for release.